website/docs/docs/build/saved-queries.md

```diff
@@ -119,6 +119,8 @@ saved-queries:
 ```
 </File>
 
+For more info on `dbt_project.yml` and config naming conventions, check out [naming convention](/reference/dbt_project.yml).
```

Comment:
"Naming convention" isn't mentioned anywhere on the ref page, so updating the description for accuracy.
```suggestion
For more information on `dbt_project.yml` and config naming conventions, see the [dbt_project.yml reference page](/reference/dbt_project.yml).
```